LibTCPServer.emx

```diff
@@ -402,7 +402,7 @@ delete job;]]></body>
 		const RTPortDescriptor* portDescriptor = actorData->ports;
 		for (int i = actorData->numPorts; --i >= 0 && !foundPortDescriptor; ++portDescriptor ) {
 			RTProtocol* port = RTObjectOf( RTProtocol, portDescriptor->offset, this );
-			if (portName == port->getName()) {
+			if (!RTMemoryUtil::strcmp(portName.c_str(), port->getName())) {
```

Comment:
@rajneshkumarmalik What is the reason for this change? There is an operator== defined on std::string for comparing with a const char* string. Was it not invoked here?

Reply:
Clang compiler was showing warning for this scenario. After searching on internet I find out that It is not supported by Clang compiler. So, this fix was made to keep Clang compiler happy. Kindly let me know your feedback.

Reply:
I got following warning.
/TCPServer.cpp:821:24: warning: result of comparison against a string literal is unspecified (use an explicit string comparison function instead) [-Wstring-compare]
                        if (port->getName() == "external" && port->getInfo()>protocol>name == "External")
                        
When I visit llvm github repo [https://github.com/llvm/llvm-project/blob/main/libcxx/include/string] I find out some comment to remove support in C++20. I think it is not present for all C++ version in clang now. I tried to debug but somehow control is not hitting this function. 
bool operator==(const charT* lhs, const basic_string<charT, traits, Allocator>& rhs) noexcept;  // removed in C++20

Reply:
I have rechecked it again. In this case warning is not observed. I might fix this code to maintain uniformity and assuming that in future we might get warning for this piece of code. Warning is observed in below case only. 
Kindly let me know that I can revert this code or keep it.

Reply:
@rajneshkumarmalik If there is no warning we shouldn't change this code. The new code is less readable than the original one, so please revert it.

Reply:
Done.